src/test_cose.rs

```diff
@@ -3,6 +3,8 @@ use util_test::{sign, verify_signature};
 use {CoseError, SignatureAlgorithm, SignatureParameters};
```

Comment:
This file is getting a little large. Maybe we should separate between COSE sign/verify tests and decoder tests. When we fix #46 the sign/verify tests should go away.

Reply:
It's just a test file, though - hopefully we won't even have to open it in the future.